client/src/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import FollowButton from './components/FollowButton.jsx';
+import $ from 'jquery';
+
+const round = {
+  borderRadius: "50%"
+};
+
+class User extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: 'JimHendrix',
+      photo: 'https://unsplash.it/120/120/?random',
+      followers: 741,
+      tracks: 4,
+      isFollowed: false
+    }
+    this.handleAddFollowClick = this.handleAddFollowClick.bind(this);
+  }
+
+  handleAddFollowClick( ) {
+    console.log('AddFollowButton was clicked');
+    if (this.state.isFollowed === true) {
+    this.setState({isFollowed: !this.state.isFollowed, followers: this.state.followers - 1});
+    $.ajax({
+      url: '/decrementFollowers',
+      type: 'PATCH',
+      data: { username: this.state.username },
+      success: (status) => {
+        console.log('This is the status inside the succes of PATCH decrement request": ', status);
+      },
+      error: (error) => {
+        console.log('This is the error from the PATCH decrement request: ', error);
+      }
+    });
+  } else {
+    this.setState({isFollowed: !this.state.isFollowed, followers: this.state.followers + 1 });
+    $.ajax({
```

Comment:
Consider abstracting the AJAX calls out into a separate function to prevent repetitive code, including the changes buts as parameters for the function like the url.

Reply:
This is a great suggestion, and I hope to incorporate this refactor after we have things up and running. Due to time constraints, I will merge as is, and come back to refactor at a later point.